### PR TITLE
chore: sample action client

### PR DESCRIPTION
**Pull request by tomer-epstein**

### CODE REVIEW

File: types/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@sap-devx/bas-platform-types",
   "publisher": "SAPOSS",
-  "version": "0.0.33",
+  "version": "0.0.13",
```
Comment (ira-gordin-sap) [inline types/package.json:4]: Is it by matter?

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
 	"name": "bas-platform",
 	"displayName": "App-Studio-Toolkit Extension",
 	"description": "SAP Business Application Studio Toolkit extension",
-	"version": "0.0.33",
+	"version": "0.0.13",
```
Comment (ira-gordin-sap) [inline package.json:5]: Is it by matter?

**Review by ira-gordin-sap — APPROVED** [review]
I hope versions changed by matter.

File: types/package.json (hunk 1 above)
Comment (tomer-epstein) [reply types/package.json:4]: Yes. Yes. Later we will decide both when to bump it to 1.0.0

File: package.json (hunk 2 above)
Comment (tomer-epstein) [reply package.json:5]: Yes. Later we will decide both when to bump it to 1.0.0

**Pull request merged**